contributors/devel/sig-release/release.md

```diff
@@ -194,57 +194,42 @@ Each release is part of a broader Kubernetes lifecycle:
 
 ![Image of Kubernetes release lifecycle spanning three releases](release-lifecycle.png)
 
-## Removal Of Items From The Milestone
-
-Before getting too far into the process for adding an item to the milestone,
-please note:
-
-Members of the [Release Team][release-team] may remove issues from the
-milestone if they or the responsible SIG determine that the issue is not
-actually blocking the release and is unlikely to be resolved in a timely
-fashion.
+# Milestone Maintainers
 
-Members of the Release Team may remove PRs from the milestone for any of the
-following, or similar, reasons:
+To maintain up-to-date status on milestone inclusion, we rely on a set of Milestone Maintainers (members of the `kubernetes-milestone-maintainers` GitHub team) to apply the appropriate labels to issues / PRs. This is facilitated by `/milestone` commands and bot automation.
 
-- PR is potentially de-stabilizing and is not needed to resolve a blocking
-  issue
-- PR is a new, late feature PR and has not gone through the enhancements
-  process or the [exception process][exceptions]
-- There is no responsible SIG willing to take ownership of the PR and resolve
-  any follow-up issues with it
-- PR is not correctly labelled
-- Work has visibly halted on the PR and delivery dates are uncertain or late
+The members of the [`milestone-maintainers`](https://github.com/orgs/kubernetes/teams/milestone-maintainers/members)
+GitHub team are entrusted with the responsibility of specifying the release milestone on GitHub artifacts by assigning a milestone label to an enhancement.
 
-While members of the Release Team will help with labelling and contacting
-SIG(s), it is the responsibility of the submitter to categorize PRs, and to
-secure support from the relevant SIG to guarantee that any breakage caused by
-the PR will be rapidly resolved.
+Members of the `kubernetes-milestone-maintainers` GitHub team, called milestone maintainer, can include:
+- SIG leadership (SIG Chairs / Technical Leads) from all SIGs
+- SIG milestone maintainers from all SIGs (in addition to SIG leadership)
```

Comment:
How I'm reading this: that SIG milestone maintainers can be milestone maintainers (given lines 201 and 204). But maybe I'm missing something nuanced?